### DISCUSSION TITLE
[DX] Increase log level for PHP/Composer auth errors

### DISCUSSION THREAD

**Discussion by benr77** [renovatebot/renovate#30216] [Suggest an Idea]
### Tell us more.

We use Renovate to update our PHP/Composer dependencies. In general this works really well thank you.

However, we also use _private_ Composer packages (hosted in Gitlab Package Manager). If we forget to allow Renovate's repository access to the new package repository, this results in many "empty commit" messages in Renovate's output, and no updates are applied until the issue is resolved. 

Only when we change the log level to `debug` do we see "Authentication to <new-package> repository failed" errors, and upon fixing the required access for Renovate all starts working again.

My suggestion is to increase the log level severity used for these "authentication failed" errors, so that debug mode is not needed to see them. If they were shown by default this would eliminate a confusing and time-consuming debugging process.

I would also consider Renovate not being able to access a Composer package repository as a serious error, so having the error only show in the debug level is clearly not appropriate.

Thanks!

**Comment by viceice** [comment]
no, use log level remap option

**Reply by rarkins** [reply]
We might want to default to warning for package lookup failures from now on, and if users want to ignore it then they can remap that message or ignore those deps?

**Reply by benr77** [reply]
@viceice Thanks - the log level remap feature is new to me and looks like it will solve my issue. 

However, it does not address the underlying issue, and if a user is unaware of this issue when it happens to them it still means length debugging to find the issue the first time. I agree with @rarkins that defaulting to `warn` for lookup/auth failures is the better and more correct solution.
